internal/request/graphql/schema/types/base.go

```diff
@@ -216,53 +216,181 @@ func FloatListOperatorBlock(op *gql.InputObject) *gql.InputObject {
+				Type:        gql.NewList(gql.NewNonNull(Float64)),
 			},
 			"_nin": &gql.InputObjectFieldConfig{
 				Description: ninOperatorDescription,
-				Type:        gql.NewList(gql.NewNonNull(gql.Float)),
+				Type:        gql.NewList(gql.NewNonNull(Float64)),
 			},
 		},
 	})
 }
 
-// NotNullFloatListOperatorBlock filter block for [NotNullFloat] types.
-func NotNullFloatListOperatorBlock(op *gql.InputObject) *gql.InputObject {
+// NotNullFloat64ListOperatorBlock filter block for [NotNullFloat] types.
+func NotNullFloat64ListOperatorBlock(op *gql.InputObject) *gql.InputObject {
 	return gql.NewInputObject(gql.InputObjectConfig{
-		Name:        "NotNullFloatListOperatorBlock",
-		Description: "These are the set of filter operators available for use when filtering on [Float!] values.",
+		Name:        "NotNullFloat64ListOperatorBlock",
+		Description: "These are the set of filter operators available for use when filtering on [Float64!] values.",
+		Fields: gql.InputObjectConfigFieldMap{
+			"_any": &gql.InputObjectFieldConfig{
+				Description: anyOperatorDescription,
+				Type:        op,
+			},
+			"_all": &gql.InputObjectFieldConfig{
+				Description: allOperatorDescription,
+				Type:        op,
+			},
+			"_none": &gql.InputObjectFieldConfig{
+				Description: noneOperatorDescription,
+				Type:        op,
+			},
+		},
+	})
+}
+
+// Float32OperatorBlock filter block for Float32 types.
+func Float32OperatorBlock() *gql.InputObject {
+	return gql.NewInputObject(gql.InputObjectConfig{
+		Name:        "Float32OperatorBlock",
+		Description: float32OperatorBlockDescription,
+		Fields: gql.InputObjectConfigFieldMap{
+			"_eq": &gql.InputObjectFieldConfig{
+				Description: eqOperatorDescription,
+				Type:        Float32,
+			},
+			"_ne": &gql.InputObjectFieldConfig{
+				Description: neOperatorDescription,
+				Type:        Float32,
+			},
+			"_gt": &gql.InputObjectFieldConfig{
+				Description: gtOperatorDescription,
+				Type:        Float32,
+			},
+			"_ge": &gql.InputObjectFieldConfig{
+				Description: geOperatorDescription,
+				Type:        Float32,
+			},
+			"_lt": &gql.InputObjectFieldConfig{
+				Description: ltOperatorDescription,
+				Type:        Float32,
+			},
+			"_le": &gql.InputObjectFieldConfig{
+				Description: leOperatorDescription,
+				Type:        Float32,
+			},
+			"_in": &gql.InputObjectFieldConfig{
+				Description: inOperatorDescription,
+				Type:        gql.NewList(Float32),
+			},
+			"_nin": &gql.InputObjectFieldConfig{
+				Description: ninOperatorDescription,
+				Type:        gql.NewList(Float32),
+			},
+		},
+	})
+}
+
+// Float32ListOperatorBlock filter block for [Float32] types.
+func Float32ListOperatorBlock(op *gql.InputObject) *gql.InputObject {
+	return gql.NewInputObject(gql.InputObjectConfig{
+		Name:        "Float32ListOperatorBlock",
+		Description: "These are the set of filter operators available for use when filtering on [Float32] values.",
```

Comment:
todo: Please move these descriptions to the descriptions.go file

Reply:
I'm not a fan of not having this inline. I'd rather be able to read it with the rest of the object declaration. We currently have a mix of both so I'm not sure we've determined an official consensus to warrant a todo here.